src/Views/scstormrunoff.html

```diff
@@ -134,10 +134,125 @@ <h4>Citation</h4>
+                                
+                                <span class="form-row">
+                                    <div class="form-group">
+                                        <label for="PeakRateFactor">Peak Rate Factor, PRF</label> <span ng-if="SyntheticUrbanHydrograph.PeakRateFactor.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                        <input type="number" name="PeakRateFactor" ng-model="vm.peakRateFactor" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                    <div class="form-group">
+                                        <label for="StandardCurveNumber">Standard Curve Number, CN</label> <span ng-if="SyntheticUrbanHydrograph.StandardCurveNumber.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                    <input type="number" name="StandardCurveNumber" ng-model="vm.standardCurveNumber" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                </span>
+                                <span class="form-row">
+                                    <div class="form-group">
+                                        <label for="WatershedRetention">Watershed Retention, S</label> <span ng-if="SyntheticUrbanHydrograph.WatershedRetention.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                        <input type="number" name="WatershedRetention" ng-model="vm.watershedRetention" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                    <div class="form-group">
+                                        <label for="InitialAbstraction">Initial Abstraction</label> <span ng-if="SyntheticUrbanHydrograph.InitialAbstraction.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                        <input type="number" name="InitialAbstraction" ng-model="vm.initialAbstraction" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                </span>
+
+                                <label for="TimeOfConcentrationMin">Time of Concentration, Tc (minutes)</label> <span ng-if="SyntheticUrbanHydrograph.TimeOfConcentrationMin.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
```

Comment:
I like that this is blank, and I am now wondering if we should all drop downs start out blank like this one does? @amedenblik , what do you think?